pkg/controller/secret/secret_controller.go

```diff
@@ -148,6 +148,9 @@ func createPagerdutyRoute() *alertmanager.Route {
 		// https://issues.redhat.com/browse/OSD-6327
 		{Receiver: receiverNull, Match: map[string]string{"alertname": "CannotRetrieveUpdates"}},
 
+		//https://issues.redhat.com/browse/OSD-6559
+		{Receiver: receiverNull, Match: map[string]string{"alertname": "PrometheusNotIngestingSamples", "namespace": "openshift-user-workload-monitoring"}},
```

Comment:
Why do we get any alerts for this namespace?  I expect we need to make sure UWM is up, but is there a specific set of alerts we should allow and discard the rest?

Reply:
We're responsible for ensuring the resources in this namespace (prom pods, thanos pods) are running and working properly. Maybe we can make a list of Alerts we care about and scope it to that. For the short term, this will generate noise for every deployment by default, so we should silence this, then reevaluate other alerts from this NS IMO.

Reply:
Ack.  Let's card up something to address this more long term but understood there's a tactical need.
/approve